src/date-picker/date-picker.jsx

```diff
@@ -149,11 +159,11 @@ const DatePicker = React.createClass({
 
   getDefaultProps() {
     return {
-      formatDate: DateTime.format,
+      formatDate: toIsoDateString,
```

Comment:
Ah, good point. I didn't grok where that originated. I read `DateTime` as being a system function.